### DISCUSSION TITLE
第4节课笔记(24班)

### DISCUSSION THREAD

**Discussion by zhjunqin** [InternLM/Tutorial#381] [笔记]
第4次课笔记(24班)

**Comment by yansuoyuli** [comment]
https://blog.csdn.net/m0_58431543/article/details/135629092

**Comment by yezhou05** [comment]
https://app.yinxiang.com/fx/c5da66c6-50aa-404b-baaf-9382d64046c6

**Comment by waurblove** [comment]
https://www.bilibili.com/opus/887229063845904388?spm_id_from=333.999.0.0

**Comment by kelenlv** [comment]
https://github.com/kelenlv/LLM_notes/blob/main/Day4.md

**Comment by lindsey-chang** [comment]
https://blog.csdn.net/qq_37397652/article/details/135718180?spm=1001.2014.3001.5501

**Comment by ljn12yyds** [comment]
https://blog.csdn.net/ljn20001229/article/details/135731280?spm=1001.2014.3001.5501

**Comment by flh1107** [comment]
https://www.bilibili.com/read/cv29920408/

**Comment by Xinyi-Cheng** [comment]
https://zhuanlan.zhihu.com/p/679344655

**Comment by ch521-od** [comment]
https://blog.csdn.net/qq_74261455/article/details/135723091?spm=1001.2014.3001.5501

**Comment by robotzm** [comment]
https://blog.csdn.net/weixin_48319544/article/details/135790403?spm=1001.2014.3001.5501
